pkg/didcomm/crypto/crypter.go

```diff
@@ -18,7 +14,7 @@ type Crypter interface {
 	// returns:
 	// 		[]byte containing the encrypted envelope
 	//		error if encryption failed
-	Encrypt(payload []byte, sender KeyPair, recipients []*[chacha.KeySize]byte) ([]byte, error)
+	Encrypt(payload []byte, sender KeyPair, recipients [][]byte) ([]byte, error)
```

Comment:
This will still need a TODO. The key type information should be included with the keys.

Reply:
done

Reply:
What is the GitHub number?

Reply:
#272

will updated comment